README.md

```diff
@@ -53,6 +53,9 @@ start showing up properly.
 Please send pull requests improving the usage and fixing bugs, improving documentation and providing
 better examples, or providing some testing, because these things are important.
 
+##Example projects using this plugin
+* [es6-library-boilerplate](https://github.com/CurtisHumphrey/es6-library-boilerplate)
```

Comment:
If I remember right there is more than one file at play here to make it all work. I could do a version if you want but part of the point of that project is to make it easy to setup coveralls with that stack so its not going away anytime sooner.